apps/src/templates/sectionProgress/ProgressBox.jsx

```diff
@@ -11,7 +11,13 @@ const styles = {
     boxSizing: 'content-box'
   },
   filler: {
-    width: 20
+    width: 20,
+    position: 'absolute'
+  },
+  lessonNumber: {
```

Comment:
I pulled this in to try it out with the Lesson Status Dialog and with just one number for the lesson number it does not center the number.

Reply:
Good catch!  I was focused on wide 2-digit numbers being too big, I forgot to think about the singles. Updating...